### DISCUSSION TITLE
New Year, New Website

### DISCUSSION THREAD

**Discussion by dbanty** [dbanty/dylananthony.com#194] [Posts]
Questions or comments about the post? Leave them here!

**Comment by ekzhang** [comment]
Thanks for the really cool blog post. Love that you're using Zola. Just a tip / recommendation to check out: I actually just recently changed my current website from using Zola to a SvelteKit static build, this has the advantage of being able to use Tailwind and SSG, while still being able to import custom Markdown + have more customizability. It's also a lot simpler than Next.js (e.g., no React), and you can opt-out of including any JavaScript in the client build by turning off `hydrate` and `router`.

The downside is that you have to deal with the NPM ecosystem of dependencies, of course, so Zola still has an advantage there. Just wanted to bring up the topic though. :)

**Reply by dbanty** [reply]
Thanks for the suggestion! I actually checked out Svelte (not sure if kit was a thing yet) a little over a year ago. At the time the JetBrains tooling was pretty hard to use so I gave up. Hopefully it's gotten better so the next time I need a more complex site I can try it out!

**Reply by ekzhang** [reply]
Yeah, it's gotten a lot better in the last year, especially SvelteKit - which is what makes the static generation possible! I also tried using SvelteKit about half a year ago and gave up (it was too unstable), but it really does work well now. :)
